Pokedex_ios/View/Controller/PokemonList/PokemonListViewController.swift

```diff
@@ -62,20 +64,20 @@ class PokemonListViewController: UIViewController {
 
     func initViewModel() {
         viewModel.getPokemons()
-
         viewModel.reloadData = { [weak self] in
             DispatchQueue.main.async {
                 self?.pokemonsTableView.reloadData()
             }
         }
+
         viewModel.showErrorAlert = {
-            let sheet = UIAlertController(title: "Something went wrong",
-                                          message: "Can't show pokemons right now, please try again later", preferredStyle: .alert)
+            let sheet = UIAlertController(title: "Something went wrong", message: "Can't show pokemons right now, please try again later",
```

Comment:
also this changes in format that you didn´t made, are you running swiftlint? do you have the pre-commit hooks setup?